tests/server/metrics-errors.js

```diff
@@ -36,18 +36,29 @@ define([
     }, dfd.reject.bind(dfd)));
   };
 
-  suite['#get /metrics-errors - returns 200 without query'] = function () {
```

Comment:
Should we keep the `GET` variant around to handle clients that loaded before the train cut and are still reporting to the old endpoint?